server/api/router.go

```diff
@@ -101,6 +101,7 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) (*mux.
 	clusterRouter.HandleFunc("/store/{id}", storeHandler.Delete).Methods("DELETE")
 	clusterRouter.HandleFunc("/store/{id}/state", storeHandler.SetState).Methods("POST")
 	clusterRouter.HandleFunc("/store/{id}/label", storeHandler.SetLabels).Methods("POST")
+	clusterRouter.HandleFunc("/store/{id}/label?force", storeHandler.SetLabelsForce).Methods("POST")
```

Comment:
em, not like this. You can refer how the force delete store was [implemented](https://github.com/pingcap/pd/blob/90b765c268076d678016de0d4b77cba1c7c2640a/server/api/store.go#L168)

Reply:
OK, I modify it as the delete store does.